templates/vue/src/components/NodeModal.vue

```diff
@@ -486,22 +521,18 @@ export default {
         }
       }
     },
+    async handlePublish() {
+      this.node.status = "publish"
+      this.handleSubmit()
+    },
     async handleDraftSubmit() {
-      this.formErrors = this.validateNode()
-      if (!this.formErrors.length) {
-        this.node.status = "draft"
-        this.updateNodeCoordinates()
-
-        if (this.node.mediaType == "url-embed" && this.node.behaviour != "embed") {
-          await this.setLinkData()
-        }
-
-        if (this.shouldReloadDuration()) {
-          this.loadDuration = true
-        } else {
-          return this.submitNode()
-        }
```

Comment:
Just checking.. was this removed on purpose? I don't see it in any other part so wanted to make sure it's not a mistake.

Reply:
This was removed because it is almost an exact duplicate of the error checking under handleSubmit(). It probably does not turn up if you cmd+F for it, because there's minute differences that set node status to draft in there.